### ISSUE TITLE
Incorporate feedback from using a generated library in an expo project

### ISSUE THREAD

**Issue by Johennes** [jhugman/uniffi-bindgen-react-native#195]
User badrb submitted some feedback on [matrix](https://matrix.to/#/!IZzRKGHTyWgRTpUJDh:matrix.org/$RxtMNzTJ1rINxjKBf9Tn607jcrohyrrun-ggI7rz3W8?via=matrix.org&via=catgirl.cloud&via=ellis.link) that we might want to incorporate into the docs.

> I managed to get it working! 🎉
>
> The main issue, apart from the TypeScript problem ⬆️, was related to my monorepo setup. This guide was super helpful: [Using Bun with Workspaces](https://bun.sh/guides/install/workspaces).
> 
> Additionally, I found that running the `bun prepare` command was mandatory for using a module from another project in my case. Including this step in the documentation's step-by-step pages would be a great improvement.
>
> Lastly, I had to adjust my `tsconfig.json` settings to the following:
>
> ```json
> {
>   "noUnusedLocals": false,
>   "noUnusedParameters": false
> }
> ```
>
> Thanks for your help 🙃!

**Comment by hassankhan** [comment]
My 2c: I think the `ubrn:ios` or `ubrn:android` package scripts should trigger the prepare script (i.e. `bob build`) after completing.

**Comment by badrbouslikhin** [comment]
Additional useful note (please correct me if I'm mistaken):  
- Struct fields cannot be accessed directly from JavaScript. For example, the following will not work:  
  ```javascript
  const property = myObjectInstance.myProperty;
  ```  
  Given the Rust code:  
  ```rust
  #[derive(uniffi::Object)]
  struct MyObject {
      my_property: u32,
  }
  ```  
  Instead, a getter method is required.

**Comment by jhugman** [comment]
> Struct fields cannot be accessed directly from JavaScript. For example, the following will not work: 

This is correct. Do you think this should be noted in the `uniffi-bindgen-react-native` docs or in the `uniffi-rs` docs?

**Comment by badrbouslikhin** [comment]
I think it should be noted in the `uniffi-bindgen-react-native` documentation, as it is targeted toward JavaScript specifically.
